app/main.js

```diff
@@ -911,6 +916,20 @@ function showNotification (text) {
   })
 }
 
+function resetBreaks () {
```

Comment:
this looks good, can you thought do this for other instances of this issue as well?  There is same problem with calling `closeWindows` when skipping to next break/microbreak. Looks like it could be refactored into function and reused in all 3 places.